stonesoup/types/orbitalstate.py

```diff
@@ -0,0 +1,840 @@
+            state_vector[4] = EclipticLongitude(state_vector[4])
+            state_vector[5] = EclipticLongitude(state_vector[5])
+
+            self.state_vector = keplerian_to_rv(
+                state_vector, grav_parameter=self.grav_parameter)
+
+        elif self.coordinates.upper() == 'TLE':
+            # TODO: ensure this works for parabolas and hyperbolas
+
+            # First enforce the correct type
+            state_vector[0] = Inclination(state_vector[0])
+            state_vector[1] = EclipticLongitude(state_vector[1])
+            state_vector[3] = EclipticLongitude(state_vector[3])
+            state_vector[4] = EclipticLongitude(state_vector[4])
+
+            # Get the semi-major axis from the mean motion
+            semimajor_axis = np.cbrt(self.grav_parameter/state_vector[5]**2)
+
+            # True anomaly from mean anomaly
+            tru_anom = tru_anom_from_mean_anom(state_vector[4], state_vector[2])
+
+            # Use given and derived quantities to convert from Keplarian to
+            # Cartesian
+            self.state_vector = keplerian_to_rv(StateVector([state_vector[2], semimajor_axis,
+                                                             state_vector[0], state_vector[1],
+                                                             state_vector[3], tru_anom]),
+                                                grav_parameter=self.grav_parameter)
+
+        elif self.coordinates.lower() == 'equinoctial':
+
+            # First enforce the correct type for mean longitude
+            state_vector[5] = EclipticLongitude(state_vector[5])
+
+            # Calculate the Keplarian element quantities
+            semimajor_axis = state_vector[0]
+            raan = np.arctan2(state_vector[3], state_vector[4])
+            inclination = 2*np.arctan(state_vector[3]/np.sin(raan))
+            arg_per = np.arctan2(state_vector[1], state_vector[2]) - raan
+            mean_anomaly = state_vector[5] - arg_per - raan
+            eccentricity = state_vector[1]/(np.sin(arg_per + raan))
+
+            # True anomaly from mean anomaly
+            tru_anom = tru_anom_from_mean_anom(mean_anomaly, eccentricity)
+
+            # Convert from Keplarian to Cartesian
+            self.state_vector = keplerian_to_rv(StateVector([eccentricity, semimajor_axis,
+                                                             inclination, raan, arg_per,
+                                                             tru_anom]),
+                                                grav_parameter=self.grav_parameter)
+        else:
+            raise TypeError("Coordinate keyword not recognised")
+
+    # Some vector quantities
+    @property
+    def _nodeline(self):
+        """The vector node line (defines the longitude of the ascending node
+        in the Primary-centered inertial frame)
+
+        Returns
+        -------
+        : numpy.array
+            The node line (defines the longitude of the ascending node)
+        """
```

Comment:
`@property` docstring is wrong format. Example can be found [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html)

Reply:
But there aren't any getter methods. So... just remove the docstrings? What if you want to know what the methods do?

Reply:
This method if the getter. I think this should be right (but double check once docs rendered):
```python
    @property
    def _nodeline(self):
        """numpy.ndarray: The vector node line (defines the longitude of the ascending node
        in the Primary-centered inertial frame)
        """
```

Reply:
Check docs. That work?